src/lib/components/primitives/button/button.story.svelte

```diff
@@ -1,18 +1,18 @@
 <script lang="ts">
   import Button from "./button.svelte";
-  import { initStyles } from '$widget/styles.store';
+  import { initialize } from '$widget/styles.store';
 
   // Button props
   export let busy = false;
-  export let customCss = [];
+  // export let customCss = [];
```

Comment:
Throughout this PR, you'll notice I'm deactivating the "runtime" customization stuff for now. We'll have a future story to implement this more maturely.